src/main/resources/wa-task-completion-employment-et_englandwales_multiple.dmn

```diff
@@ -21,6 +21,19 @@
           <text>"Auto"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_00owiss">
+        <inputEntry id="UnaryTests_0l84new">
+          <text>"preAcceptanceCase",
```

Comment:
AFAIK, preAcceptanceCase is not an event on Multiples. So no need to have it here. It was probably a copy and paste from the singles.